bugbug/similarity.py

```diff
@@ -139,6 +143,15 @@ def evaluation(self):
                 num_hits = 0
                 queries += 1
                 similar_bugs = self.get_similar_bugs(bug)[:10]
+                if end_to_end:
+                    sim_bugs = bugzilla.get(similar_bugs)
```

Comment:
We should have these bugs in the DB, so we shouldn't need to retrieve them from bugzilla